static/js/components/SourceList.jsx

```diff
@@ -8,17 +8,31 @@ const SourceList = ({ sources }) => (
     <h2>
 Sources
     </h2>
-    <ul>
+
+    <table id="tab">
+     <tr>
+     <th>Last detected</th><th>Name</th><th>RA</th><th>DEC</th><th>varstar</th><th>transient</th><th>disagree</th><th>Score</th><th>N detections</th><th>Simbad Class</th></tr>
       {
         sources.map((source, idx) => (
-          <li key={source.id}>
-            <Link to={`/source/${source.id}`}>
-              {source.id}
-            </Link>
-          </li>
+          <tr key={source.id}>
+            <td>{source.last_detected.split(".")[0]}&nbsp;&nbsp;</td>
+            <td><Link to={`/source/${source.id}`}>
+              {source.id}</Link></td>
+              <td>{Number(source.ra_dis).toFixed(3)}</td><td>{Number(source.dec_dis.toFixed(4))}</td>
+              <td>{source.varstar.toString()}</td>
+              <td>{source.transient.toString()}</td>
+              <td>{(source.transient == source.varstar).toString()}</td>
+              <td>{Number(source.score).toFixed(2)}
+              </td>
+              <td>{source.detect_photometry_count}</td>
+              <td>{source.simbad_class}</td>
+          </tr>
         ))
       }
-    </ul>
+    </table>
+    <script>
+    sorttable.makeSortable(document.getElementById('tab'));
```

Comment:
What does this do?

Reply:
I was trying to add a sortable table, but failed. Removed this.